public/javascript/atom/atom.html

```diff
@@ -11,16 +11,19 @@
         <div class="col-md-6 asset-data">
             <h1 class="atom-section" id="section">Atom Data</h1>
 
-            <label ng-if="!publishedAtom" for="embedLink">Link to capi:</label>
-            <label ng-if="publishedAtom" for="embedLink">Embed link:</label>
+            <label for="embedLink">Link to capi:</label>
             <div id="embed-link">
-                <a
+            <a
                     href="{{linkToCapi}}"
```

Comment:
Though its not part of this PR, this should be [`ng-href`](https://docs.angularjs.org/api/ng/directive/ngHref).